### PR TITLE
CI: Make sure oc/podman env command works

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

**Comment by praveenkumar** [comment]
depends on #3585

File: .github/workflows/macos-microshift.yml [hunk 1]
```diff
@@ -50,3 +50,9 @@ jobs:
         run: sleep 10
       - name: Run crc with microshift preset
         run: crc start --pull-secret-file dummy_pull.json
+      - name: check if podman binary is symlinked correctly
+        run: |
+          eval $(crc podman-env) && podman pull quay.io/crc-org/crc-extension:latest
+      - name: check if oc binary is symlinked correctly
+        run: |
+          eval $(crc oc-env) && oc --kubeconfig ${HOME}/.crc/machines/crc/kubeconfig get pods -A
```
Comment (cfergeau) [inline .github/workflows/macos-microshift.yml]: we don't need to specify the path to the kubeconfig file.

Comment (praveenkumar) [reply .github/workflows/macos-microshift.yml]: @cfergeau yes, not any more. Will update this PR.

Comment (praveenkumar) [reply .github/workflows/macos-microshift.yml]: @cfergeau PR is now updated with removed `kubeconfig` path.

**Pull request merged**